system/bluetooth_monitor/schema/bluetooth_monitor.shcema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for bluetooth monitor",
+  "type": "object",
+  "definitions": {
+    "bluetooth_monitor": {
+      "type": "object",
+      "properties": {
+        "address": {
+          "type": "array",
+          "description": "Bluetooth addresses of the device to monitor",
+          "items": {
+            "type": "string"
+          },
+          "default": ["4C:B9:9B:6E:7F:9A"]
+        },
+        "port": {
+          "type": "integer",
+          "description": "Port to connect to on the device",
```

Comment:
@YuqiHuai This value is not for the port to connect to the bluetooth device. This is the port number to connect to L2ping service.
Please see [Block diagram](https://github.com/YuqiHuai/autoware.universe/tree/refactor/bluetooth_monitor/system/bluetooth_monitor#block-diagram)

Reply:
Hi @ito-san , thank you for matching this mistake! I have just corrected it.